src/app/api/collections/tree.py

```diff
@@ -36,31 +36,38 @@ def build_hierarchy(
         tree_hierarchy.update({str(collection.parent_id): []})
 
     tree_hierarchy[str(collection.parent_id)].append(portal_node)
-    tree_hierarchy[str(collection.noderef_id)] = portal_node.children
+    tree_hierarchy[str(collection.node_id)] = portal_node.children
     return tree_hierarchy
 
 
 def tree_search(node_id: uuid.UUID) -> Search:
     s = Search().base_filters().query(qbool(filter=qterm(qfield="path", value=node_id)))
     s = s.source(
-        ["nodeRef.id", "properties.cm:title", "collections.path", "parentRef.id"]
+        [
```

Comment:
:+1:

Reply:
E.g. here it looks weird, that `properties:cm:title` is now names `COLLECTION_TITLE`, the `COLLECTION_` prefix would suggest that it is the title of a collection which would be different from the title of a material.

Reply:
See ElasticResourceAttribute descriptor - this is indeed ultra confusing.